src/org/zaproxy/zap/extension/formHandler/resources/Messages.properties

```diff
@@ -0,0 +1,22 @@
+# This file defines the default (English) variants of all of the internationalised messages
+
+formHandler.options.title										= Form Handler
+formHandler.options.desc										= This extension allows a user to change the default values used by ZAP Spiders.
+formHandler.options.label.description							= <html> <body> This Form Handler extension allows for custom configuration of values for the Spider to use based on field names. Newly created field names must match the field name in the form being processed. The field name inputted is not case sensitive, however the values are and will be reflected in the POST form. If only a field name is provided the Spider will use an empty string as a value. If a form's field does not match any in the extension then it will be passed to the Default Value Generator, which may not provide proper values. </body> </html>
+formHandler.options.table.column.enabled						= Enabled
+formHandler.options.table.column.field							= Field Name
+formHandler.options.table.column.value							= Value
+formHandler.options.dialog.token.field.label.name  				= Name:
+formHandler.options.dialog.token.field.label.value 				= Value:
+formHandler.options.dialog.token.field.label.enabled			= Enabled:
+formHandler.options.dialog.token.warning.name.repeated.title	= Duplicated Field Name
+formHandler.options.dialog.token.warning.name.repeated.text		= A token with the given field name already exists
```

Comment:
Could you also remove the extra a white space, there's no need to try and align the equals sign or whatever...